docs/references/ic-interface-spec.md

```diff
@@ -2121,7 +2121,9 @@ The optional `settings` parameter can be used to set the following settings:
 
 -   `freezing_threshold` (`nat`)
 
-    Must be a number between 0 and 2<sup>64</sup>-1, inclusively, and indicates a length of time in seconds.
+    Must be a number between 86400 (or equivalent of 1 day in seconds) and 2<sup>64</sup>-1, inclusively, and indicates a length of time in seconds.
+
+    Values between 1 day and 2 weeks, or more formally any value that is at least 1 day and less than 14 days, can only be set if the special `allow-unsafe-settings` is used in the relevant management canister APIs.
```

Comment:
From the discussion in the Interface Spec meeting: how about introducing a "hard" lower bound of one week and dropping the "soft" lower bound (and the additional fields `allow_unsafe_settings`)?

Reply:
Sounds good to me. I'll update accordingly.

Reply:
Done.

Reply:
Thank you!